### PR TITLE
Relayout Bottom Toolbar

### PR DESCRIPTION
**Pull request by Noordfrees**
image

It now shows the park rating, and those widgets are clickable to bring up the respective window. Also contains a commit that fixes a crash related to message pluralization – bugs always seem to be discovered directly after the feature that adds them is merged ;)

### CODE REVIEW

File: src/park_management_gui.cpp [hunk 1]
```diff
@@ -41,6 +41,7 @@ class ParkManagementGui : public GuiWindow {
  * @ingroup gui_group
  */
 enum ParkManagementWidgets {
+	/* The tabbutton constants must be in sync with #ParkManagementGuiTabs. */
```
Comment (Alberth289346) [inline src/park_management_gui.cpp]: If you add such comments in the doxygen text above, it becomes visible in the documentation as well. Also, stuff like "#ParkManagementGuiTabs" make more sense then.

Not sure if you did that, but add such a note in ParkManagementGuiTabs as well pointing to here.

Of course if you really want to be sure it should be enforced or checked, but that may be a bit too much here.

File: src/toolbar_gui.cpp [hunk 2]
```diff
@@ -358,10 +364,18 @@ Point32 BottomToolbarWindow::OnInitialPosition()
 	return pt;
 }
 
+void BottomToolbarWindow::SetTooltipStringParameters(BaseWidget *tooltip_widget) const
```
Comment (Alberth289346) [inline src/toolbar_gui.cpp:367]: misses documentation

Comment (Alberth289346) [reply src/toolbar_gui.cpp:367]: Hmm, maybe it's documented in a base class. Not too awake currently :(

File: src/toolbar_gui.cpp [hunk 3]
```diff
@@ -381,14 +395,37 @@ void BottomToolbarWindow::SetWidgetStringParameters(WidgetNumber wid_num) const
...
-	BaseWidget *w = this->GetWidget<BaseWidget>(wid_num);
-	if (pos.x < w->pos.width - w->pos.height) {
-		_inbox.DismissDisplayMessage();
-	} else {
-		_inbox.display_message->OnClick();
+		default:
+			break;
 	}
+
+	return GuiWindow::OnClick(wid_num, pos);
```
Comment (Alberth289346) [inline src/toolbar_gui.cpp]: Euhm, why do you return a value when line 396 says "void"?

**Comment by Alberth289346** [comment]
Looks nothing major, so please fix and merge, unless you think I need to check again.

**Comment by Noordfrees** [comment]
Thanks :)

**Pull request merged**